install/run-tests.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+# Script used to execute all the swiftly-install tests found in install/tests.
+# This should be run from the install directory of the repository.
+# WARNING: these tests make changes to the local filesystem and are intended to be run in a containerized environment.
+
+source ./test-util.sh 
```

Comment:
Is it possible to run tests without messing up your current swiftly install?

Reply:
We could set `SWIFTLY_HOME_DIR` to a temporary directory, but these tests also exercise the default installation path, which inevitably messes with the system install. I think it's worth it to have the tests modify the local system in order to avoid the risk of missing test coverage, especially since they're relatively easy to just run on a fresh docker container.